pyEpiabm/pyEpiabm/tests/test_sweep/test_host_progression_sweep.py

```diff
@@ -62,6 +63,29 @@ def test_neg_latent_time(self, mock_choose):
             test_sweep = pe.sweep.HostProgressionSweep()
             test_sweep._set_latent_time(self.person1, current_time)
 
+    def test_set_infectiousness(self):
+        """Tests that the set infectiousness function returns a positive
+        float.
+        """
+        # Test with person1 as InfectASympt infection status
+        self.person1.update_status(InfectionStatus.InfectASympt)
+        test_sweep = pe.sweep.HostProgressionSweep()
+        test_sweep._set_infectiousness(self.person1)
+        self.assertIsInstance(self.person1.infectiousness, float)
+        self.assertTrue(0 <= self.person1.infectiousness)
+        # Test with person1 as InfectMild infection status
+        self.person1.update_status(InfectionStatus.InfectMild)
+        test_sweep = pe.sweep.HostProgressionSweep()
```

Comment:
Why are we redefining this? Does it not work with the original `test_sweep` object?